### ISSUE TITLE
Potential inconsistency between CLI and "queryFormat: 'graphql'"

### ISSUE THREAD

**Issue by linonetwo** [rubensworks/graphql-to-sparql.js#19]
After a workaround on JSON-LD context, I can get this on cli:

```bash
$ graphql-to-sparql //Desktop/aaa.json "{ contains }"
SELECT ?contains WHERE { _:b1 <http://www.w3.org/ns/ldp#contains> ?contains. }
```
Which works with:
```js
myEngine
  .query(`
  SELECT ?contains WHERE { _:b1 <http://www.w3.org/ns/ldp#contains> ?contains. }
  `, context)
  .then(function(result) {
    (result as any).bindingsStream.on('data', function(data) {
      console.log(data.toObject());
    });
  });
// "?contains": Object { id: "https://new-japanese-concise-tutorial.solid.authing.cn/public/textbook/new-japanese-concise-tutorial-chapter-1-text.txt" }
```
But this still doesn't works:
```js
myEngine
  .query(`{ contains }`, context) // with queryFormat: 'graphql', in context
  .then(function(result) {
    return bindingsStreamToGraphQl((result as any).bindingsStream, context);
  })
  .then(console.log)
  .catch(console.warn);
// Array []
```

I'm confused by this inconsistency, how can I debug it?

**Comment by rubensworks** [comment]
Those queries are indeed equivalent. It looks like all GraphQL queries are failing in Comunica, I'll look into it.

**Comment by linonetwo** [comment]
And I tried [your suggestion](https://github.com/rubensworks/graphql-to-sparql.js/issues/17#issuecomment-457083459)

Got SPARQL result from cli, which works when query in SPARQL mode.

But query `{ ... on Container {contains} }` in GraphQL mode returns
```js
(1) […]
     0: Object {  }
     length: 1
     <prototype>: Array []
```

**Comment by rubensworks** [comment]
Ok, this seems to be the same problem as comunica/comunica#361

I know the cause, will push a fix soon.

**Comment by rubensworks** [comment]
Fixed in Comunica 1.5.3.

**Comment by linonetwo** [comment]
That was so fast!

It works, fetching data from SoLiD becomes so easy now!
